GOVERNANCE.md

```diff
@@ -5,121 +5,116 @@ project will manage this repository.
 
 ## Meetings
 
-In order to provide equitable rights to all members,
-the following process will be followed:
-
-* Official conference calls will be announced at least a week in advance.
-* Official face-to-face meetings will be announced at least 4 weeks in
-  advance.
-* Proposed changes to any document will be done via a Pull Request (PR).
-* PRs will be reviewed during official meetings, but off-line reviews
-  (LGTMs, NOT LGTMs) and comments are strongly encouraged to gauge the
-  group's opinion on the proposed change prior to the meeting.
-* During meetings, priority will be given to PRs that appear to be ready for
-  a vote over those that appear to require discussions.
-* PRs should not be merged if they have had substantial changes made within
-  two days of the meeting.
-  Rebases, typo fixes, etc. do not count as substantial.
-  Note, administrivia PRs that do not materially modify output documents
-  may be processed by admins as needed.
-* Resolving PRs ("merging" or "closing with no action") will be done as a
-  result of a motion made during a meeting, and approved.
-* Reopening a PR can be done if new information is made available, and a
-  motion to do so is approved.
-* Any motion that does not have "unanimous consent" will result in a formal
+In order to provide equitable rights to all members, the following process will
+be followed:
+
+- Official conference calls will be announced at least a week in advance.
+- Official face-to-face meetings will be announced at least 4 weeks in advance.
+- Proposed changes to any document will be done via a Pull Request (PR).
+- PRs will be reviewed during official meetings, but off-line reviews (LGTMs,
+  NOT LGTMs) and comments are strongly encouraged to gauge the group's opinion
+  on the proposed change prior to the meeting.
+- During meetings, priority will be given to PRs that appear to be ready for a
+  vote over those that appear to require discussions.
+- PRs should not be merged if they have had substantial changes made within two
+  days of the meeting. Rebases, typo fixes, etc. do not count as substantial.
+  Note, administrivia PRs that do not materially modify output documents may be
+  processed by admins as needed.
+- Resolving PRs ("merging" or "closing with no action") will be done as a result
+  of a motion made during a meeting, and approved.
+- Reopening a PR can be done if new information is made available, and a motion
+  to do so is approved.
+- Any motion that does not have "unanimous consent" will result in a formal
   vote. See [Voting](#voting).
 
 ## Membership
 
-There are three categories of project membership:
```

Comment:
fix.

Reply:
did you have to manually fix it? That concerning if so

Reply:
yes I did. The tool tried the best it could.

Reply:
that's unfortunate because then we can't automate things.
I wonder if there's a better way to write that line so the tool will like it....

Reply:
you can turn write off.

Reply:
I updated the line to be valid markdown.